data.md

```diff
@@ -185,6 +185,17 @@ The `#width` function returns the bit-width of a given `IValType`.
     rule #pow (i64) => 18446744073709551616
 ```
 
+```k
```

Comment:
Please add documentation before this code block.

We should probably never have a code block that doesn't have documentation before it.